src/components/Adyen/Adyen.js

```diff
@@ -302,29 +301,39 @@ const Adyen = ({
       isMyAccount,
       paymentMethodsType
     );
+
     if (responseData?.id) {
       createDropInInstance(responseData, paymentMethodsType);
     }
   };
 
   const generateDropIns = () => {
-    const { paymentMethods } = publisherConfig;
+    const availablePaymentMethods = getAvailablePaymentMethods(
+      publisherPaymentMethods
+    );
+
+    if (!availablePaymentMethods.length) {
+      // use function below to handle case when session returns error -
+      // e.g publisher has only iDeal but you are not using VPN
+      setGeneralError(t('Payment methods not available'));
+      setIsLoading(false);
+      return;
+    }
 
-    if (isMyAccount || totalPrice === 0) {
-      const shouldCreateBankPaymentSession = paymentMethods.some(
+    if (totalPrice === 0) {
```

Comment:
did you check the myaccount case? there the totalPrice is unavailable but we still wants to present bank payment methods there

Reply:
there should be this first condition with `isMyAccount`, my mistake when merging